### ISSUE TITLE
CLI - Render to File

### ISSUE THREAD

**Issue by SchmidL** [CloudCompare/CloudCompare#1431]
**Describe the feature you would like**
Add the "Render to File" feature as a command-line command. The viewpoint can be initially set in the GUI and the respective parameters can be entered as an argument. I really like the way CC renders the point cloud and would be amazing to have the feature also available for some automated process or esp. when reprocessing.

**Is your feature request related to a problem? Please describe.**
When multiple point clouds have to be exported, or esp. re-exported after editing of a point cloud is super cumbersome to set all the views again and save it to file.
With the CLI feature, it would be possible to store the viewpoints and just render the scenes in a batch script.

**Additional context**
I understand that this features cannot be run in -SILENT mode due to the dependencies on the UI etc.

**Comment by wsAndy** [comment]
`-RASTERIZE -GRID_STEP ` is already retained.

**Comment by dgirardeau** [comment]
Yes, and regarding the original ask, sadly there's no OpenGL support in command line (on purpose). Therefore it's currently not possible to render a cloud in this mode...

**Comment by SchmidL** [comment]
Sorry for digging it up again. but would there be an option which over CLI loads all the parameter, like the point cloud, the name of the SF and a saved Viewport and render it with a specific resolution?

**Comment by dgirardeau** [comment]
So it's still not possible with the command line, but I believe it is with [CloudCompy ](https://github.com/CloudCompare/CloudComPy) (there's maybe even an example in their documentation, or in an 'issue' where someone asked for the same thing).
